### PR TITLE
Create endpoint to create modules (Need Testing)

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/routes/module/module.router.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+
+import { moduleController } from '../../controllers';
+import { checkAccessTokenMiddleware, checkIsAdmin } from '../../middleware';
+
+const router = Router();
+
+router.use(checkIsAdmin, checkAccessTokenMiddleware);
```
Comment (VictorKmin) [inline src/routes/module/module.router.ts]: We need to check token first. Then we need to check isUserAdmin. 
In checkIsAdmin you get a role from req.user and you set it in checkAccessTokenMiddleware

File: src/validators/moduleValidator.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as Joi from 'joi';
+
+export const moduleValidator  = Joi.object().keys({
+  label: Joi.string().max(255).trim().required(),
+  description: Joi.string().max(255).trim().required(),
```
Comment (VictorKmin) [inline src/validators/moduleValidator.ts]: Maybe description and tags shouln not be required?

File: src/routes/module/module.router.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/routes/module/module.router.ts]: fixed

File: src/validators/moduleValidator.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/validators/moduleValidator.ts]: fixed

**Pull request merged**